hyperspace/primitives/src/lib.rs

```diff
@@ -370,10 +370,25 @@ pub trait MisbehaviourHandler {
 	) -> Result<(), anyhow::Error>;
 }
 
+/// Provides an interface for syncing light clients to the latest state
+#[async_trait::async_trait]
+pub trait LightClientSync {
```

Comment:
We need to add the docs to reflect the inclusion of this trait.

Reply:
Done